services/src/test/java/org/jfrog/artifactory/client/SearchTests.java

```diff
@@ -36,13 +39,21 @@ public void setup() {
 
         InputStream inputStream = this.getClass().getResourceAsStream("/sample.zip");
         assertNotNull(inputStream);
-        final String targetPath = "com/example/" + artifactId + "/1.0.0/" + artifactId + "-1.0.0-zip.jar";
+        String targetPath = "com/example/" + artifactId + "/1.0.0/" + artifactId + "-1.0.0-zip.jar";
         File deployed = artifactory.repository(localRepository.getKey()).upload(targetPath, inputStream)
                 .withProperty(colorsProp, "red", "green", "blue")
                 .withProperty(buildProp, "28")
                 .withProperty(releasedProp, true)
                 .doUpload();
         assertNotNull(deployed);
+
+        for (int i = 2; i <= 3; i++) {
+            inputStream = this.getClass().getResourceAsStream("/sample.txt");
```

Comment:
This input steam needs to be closed after it is used.
The best way to do this, is using "try with resources", they way it is demonstrated in the README:
```
     try (InputStream inputStream = Files.newInputStream(Paths.get("fileToUpload.txt"))) {
         ...
     }
```